guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -162,6 +162,7 @@
         "TEXT_USER_LEFT"                  : "{USERNAME} has left the connection.",
         "TEXT_RECONNECT_COUNTDOWN"        : "Reconnecting in {REMAINING} {REMAINING, plural, one{second} other{seconds}}...",
         "TEXT_FILE_TRANSFER_PROGRESS"     : "{PROGRESS} {UNIT, select, b{B} kb{KB} mb{MB} gb{GB} other{}}",
+        "TEXT_CLIPBOARD_AWAITING_FOCUS"   : "Focus here to view clipboard data...",
```

Comment:
"Focus" seems like an excessively technical term here: one that people not fluent in web technologies might not understand. If the required action is to hover over the clipboard, it should probably just say that plainly.